The warnig flag isn't recognized for older GCC versions (blackfin),
so just disable it.

Signed-off-by: Gustavo Zacarias <[hidden email]>

diff -Nura graphite2-1.3.3.orig/src/CMakeLists.txt graphite2-1.3.3/src/CMakeLists.txt
--- graphite2-1.3.3.orig/src/CMakeLists.txt 2015-09-24 10:06:28.877851596 -0300
+++ graphite2-1.3.3/src/CMakeLists.txt 2015-09-24 10:06:48.201519767 -0300
@@ -111,9 +111,6 @@
         COMPILE_FLAGS   "-Wall -Wextra -Wno-unknown-pragmas -Wendif-labels -Wshadow -Wctor-dtor-privacy -Wnon-virtual-dtor -fno-rtti -fno-exceptions -fvisibility=hidden -fvisibility-inlines-hidden -fno-stack-protector"
         LINK_FLAGS      "-nodefaultlibs ${GRAPHITE_LINK_FLAGS}"
         LINKER_LANGUAGE C)
-    if (CMAKE_COMPILER_IS_GNUCXX)
-        add_definitions(-Wdouble-promotion)
-    endif (CMAKE_COMPILER_IS_GNUCXX)
     if (${CMAKE_CXX_COMPILER} MATCHES  ".*mingw.*")
         target_link_libraries(graphite2 kernel32 msvcr90 mingw32 gcc user32)
     else (${CMAKE_CXX_COMPILER} MATCHES  ".*mingw.*")
